README.md

```diff
@@ -1,20 +1,24 @@
 # HacktoberfestChecker-cli
-Simple Python3 Script for checking your project in the Hacktoberfest Challenge.
+It is a simple Python3 Script for checking the number of Pull Requests you have made in the Hacktoberfest Challenge.
 
 ## How to get started
-1. clone this repository onto your local environment:
+1. Clone this repository onto your local environment:
 ```
 git clone https://github.com/kerk12/HacktoberfestChecker-cli
 ```
-2. install the dependencies
+2. Install the dependencies:
 ```
 pip install -r requirements.txt
 ```
-3. Run the app
+3. Move into the directory
+```
+cd /HacktoberfestChecker-cli
+```
+3. Run the script:
 ```
 python hfc.py
 ```
-4.Enter your or any github username
+4. Enter your github username (Enter your Username in inverted commas):
```

Comment:
What do you mean by "inverted commas"?

Reply:
I meant like "AtriTripathi". Because, maybe I might be wrong, but it didn't work for me when I gave the username directly.
Just like this.
![edit](https://user-images.githubusercontent.com/25243472/31911153-5f26d74e-b85d-11e7-925f-a87efd5eb861.jpeg)

Reply:
I wasn't able to replicate the error on my end. Please remove this line, recommit, and should the error reappear, please open an issue. Thank you for your help ;).